curriculum/js1/7.js

```diff
@@ -5,8 +5,16 @@
  * @returns {boolean}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, i = 2) => {
+  if(a === 2) return true;
+ 
+  if(a < 2 || i === a) return false;
+
+  if(a % i === 0) return false;
+
+  if(a % i !== 0) return true;
```

Comment:
wrong solution. 
numbers can only be divided by 1 and itself, so all the other numbers should not be divisible. 

```
solution = (a, i=2) => {
  if(a <= 1) return false;
  if(i===a) return true;
  if(a % i === 0) return false;
  return solution(a, i+1);
}
```